src/main/java/uk/gov/hmcts/reform/sscs/service/DwpAddressLookupService.java

```diff
@@ -32,7 +32,7 @@
 @Service
 @Slf4j
 public class DwpAddressLookupService {
-    //FIXME: Now we are onboarding HMRC, we need to replace all references to DWP with OGD (Other Government Departments)
+    //FIXME: Now we are onboarding HMRC, we need to replace all references to FTA with OGD (Other Government Departments)
```

Comment:
lol

Reply:
Don't actually think this comment is correct now. It should say "//FIXME: Now we are onboarding HMRC, we need to replace all references to DWP with FTA (First Tier Agency)". The log messages on lines 48 and 71. I changed those a few weeks ago when we thought it was going to be called 'OGD' but they since decided on FTA. Mind updating as part of this PR?